### ISSUE TITLE
[Bug]: Fails to load

### ISSUE THREAD

**Issue by TheOfficialDroid** [igorkis-scrts/A3-Antistasi-Plus#59]
### Describe the bug

Honestly, I have no idea how to describe this besides saying everything launches somewhat fine until the mission gets to the Loadout stage and fails out and hangs; We're not sure why its happening, because it was working perfectly fine an hour or so before.

### How to reproduce

I'm not even sure you can.

### Version

1.7.3

### Have you altered the code?

No

### Map

Altis

### What server?

Private dedicated server

### Time bug occured (Server time/UTC)

19/08/2021 // 20/08/2021

### Mods

```html
RHS, ACE, TFAR
```

### Additional context

For some reason I can't get the server to produce solid log files; The best I can provide are what are thrown in the console albeit lacking for the most part.

```"152.203: [Antistasi] | INFO | Starting Persistent Load."
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Warning Message: No entry 'bin\config.bin/CfgVehicles.rhsusf_mine_m14_ammo'.
Warning Message: No entry '.ammo'.
Warning Message: '/' is not a value
Weather was forced to change
Error in expression <vate _garrison = +(_x select 1);
{

if (_x find "loadouts_rebel" != 0) then { co>
  Error position: <_x find "loadouts_rebel" != 0) then { co>
  Error Undefined variable in expression: _x
File mpmissions\__cur_mp.Altis\functions\Save\fn_loadStat.sqf..., line 151
"default/BIS_fnc_log: [BIS_fnc_preload] ----- Scripts initialized at 27306 ms -----"
```

**Comment by TheOfficialDroid** [comment]
Full log for convenience; This also happens on a locally setup server as well ass my box.
[default_server_log.txt](https://github.com/igorkis-scrts/A3-Antistasi-Plus/files/7022493/default_server_log.txt)

**Comment by igorkis-scrts** [comment]
Execute this:
```
private _garrisonData = ["garrison"] call A3A_fnc_returnSavedStat;
{
    private _units = (_x#1) select {!isNil "_x"};
    _x set [1, _units];
} forEach _garrisonData;
["garrison",_garrisonData] call A3A_fnc_setStatVariable;
```
and reload the mission.

**Comment by Oranges1016** [comment]
> Full log for convenience; This also happens on a locally setup server as well as my box.
> [default_server_log.txt](https://github.com/igorkis-scrts/A3-Antistasi-Plus/files/7022493/default_server_log.txt)
> 
> Edit:
> 
> I made an edit to the mission and disabled the loading of rebel loadouts and this allowed the mission to initialize successfully, I'm not 100% sure what item was saved in one of the loadouts; Myself and a friend are making the assumption it was the M114 Explosive Charge from RHS, but we're not 100% sure.

Hi, experiencing exact same scenario down to fuckin identical .rpt logs, been fucking with arma for like 2 years now, but i dont have much experience with the EDEN editor. I've got the mission loaded into the editor, but im not sure how to disable the rebel loadouts. Is it a setting I can find?

**Comment by igorkis-scrts** [comment]
Fix is already done and will be released with 1.7.5 update.

**Comment by igorkis-scrts** [comment]
Fixed in 1.7.5.
